app/repositories/events/EventsConfigRepository.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories.events
+
+import config.MicroserviceAppConfig
+import factories.UUIDFactory
+import model.persisted.eventschedules._
+import net.jcazevedo.moultingyaml._
+import net.jcazevedo.moultingyaml.DefaultYamlProtocol._
+import org.joda.time.{ LocalDate, LocalTime }
+import org.joda.time.format.DateTimeFormat
+import play.api.Play
+import resource._
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.io.Source
+
+case class EventConfig(
+                  eventType: String,
+                  description: String,
+                  location: String,
+                  venue: String,
+                  date: LocalDate,
+                  capacity: Int,
+                  minViableAttendees: Int,
+                  attendeeSafetyMargin: Int,
+                  startTime: LocalTime,
+                  endTime: LocalTime,
+                  resourceRequirements: Map[String, Int],
+                  sessions: List[Session]
+                )
+
+object EventConfig {
+  implicit def configToEvent(c: EventConfig): Event = {
+    Event(UUIDFactory.generateUUID(),
+      EventType.withName(c.eventType),
+      c.description,
+      Location(c.location),
+      Venue(c.venue, ""),
+      c.date,
+      c.capacity,
+      c.minViableAttendees,
+      c.attendeeSafetyMargin,
+      c.startTime,
+      c.endTime,
+      c.resourceRequirements,
+      c.sessions)
+  }
+}
+
+object EventConfigProtocol extends DefaultYamlProtocol {
+  implicit object LocalDateYamlFormat extends YamlFormat[LocalDate] {
+    def write(x: LocalDate) = YamlDate(x.toDateTimeAtStartOfDay)
+    def read(value: YamlValue) = value match {
+      case YamlDate(x) => x.toLocalDate
```

Comment:
Instead of x I would write something more descriptive. Actually I am not sure what is x unless I go inside YamlDate declaration.